serializers/avro/src/main/java/io/pravega/schemaregistry/serializer/avro/impl/AvroSerializer.java

```diff
@@ -27,30 +26,32 @@
 import java.io.OutputStream;
 
 public class AvroSerializer<T> extends AbstractSerializer<T> {
```

Comment:
should we also look for an opportunity to optimize it for deserializers?
and is there an opportunity to optimize for json and protobuf serializers and deserializers too?

Reply:
Hi, @shiveshr ,

Yes, I think after optimizing serializer for avro format and measuring positive impact, we can look on on avro deserializer,
but I prefer to do it in separate tickets because if  this avro serializer fix seems rather simple, avro deserializer requires deeper investigation.

For json and protobuf first it seems that we should measure current performance, profile omb with these formats support and if there is need for improvement and there are some points for improvement do it in separate tickets.

--
Thanks,
Maria